packages/cli-e2e/__tests__/atomic.specs.ts

```diff
@@ -10,6 +10,8 @@ import {jwtTokenPattern} from '../utils/matcher';
 import {EOL} from 'os';
 import {DummyServer} from '../utils/server';
 import {loginWithApiKey} from '../utils/login';
+import {existsSync, ensureFileSync} from 'fs-extra';
+import {join} from 'path';
```

Comment:
should it be node:path ?

Reply:
could be, but the test aren't ESM so it's not that important